apps/E2E/src/RadioGroup/pages/RadioGroupPageObject.ts

```diff
@@ -24,52 +24,36 @@ class RadioGroupPage extends BasePage {
   /******************************************************************/
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
-  // Get RadioButton's accessibilityLabel
-  async getRBAccessibilityLabel(radioButtonSelector: RadioButtonSelector): Promise<string> {
-    return await (await this.getRadioButton(radioButtonSelector)).getAttribute('Name');
-  }
 
   /* This resets the RadioGroup selection by clicking/selecting the 1st RadioButton in the RadioGroup.
    * Useful in beforeEach() hooks to reset the RadioGroup before additional tests */
   async resetRadioGroupSelection(): Promise<void> {
-    await (await this._firstRadioButton).click();
-  }
-
-  async getRadioButtonAccesibilityRole(): Promise<string> {
-    return await (await this._firstRadioButton).getAttribute('ControlType');
-  }
-
-  async isRadioButtonSelected(radioButtonSelector: RadioButtonSelector): Promise<boolean> {
-    return await (await this.getRadioButton(radioButtonSelector)).isSelected();
-  }
-
-  async clickRadioButton(radioButtonSelector: RadioButtonSelector): Promise<void> {
-    await (await this.getRadioButton(radioButtonSelector)).click();
+    await (await this.getRadioButton(RadioButton.First)).click();
   }
 
-  async waitForRadioButtonSelected(radioButtonSelector: RadioButtonSelector, timeout?: number): Promise<void> {
-    await browser.waitUntil(async () => await this.isRadioButtonSelected(radioButtonSelector), {
-      timeout: timeout ?? this.waitForUiEvent,
-      timeoutMsg: 'RadioButton was not selected correctly.',
-      interval: 1000,
-    });
+  async isRadioButtonSelected(selector: RadioButton): Promise<boolean> {
+    return await (await this.getRadioButton(selector)).isSelected();
```

Comment:
You mentioned in a different PR that `isSelected()` didn't work on UWP. Is it the same here? Or does it work here?

Reply:
I can't confirm anything. The page is inaccessible on the UWP test app, and both RadioGroups are not part of the tests run on UWP. For now, this should not be an issue.